ironic/conf/drac.py

```diff
@@ -21,7 +21,14 @@
                min=1,
                help=_('Interval (in seconds) between periodic RAID job status '
                       'checks to determine whether the asynchronous RAID '
-                      'configuration was successfully finished or not.'))
+                      'configuration was successfully finished or not.')),
+    cfg.IntOpt('query_management_config_job_status_interval',
+               default=120,
+               min=1,
+               help=_('Interval (in seconds) between periodic management job '
+                      'status checks to determine whether the asynchronous '
+                      'MANAGEMENT configuration was successfully finished '
```

Comment:
Can you make Management lower case in the line above?

Reply:
Done